AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_MeshAdded.py

```diff
@@ -173,7 +173,7 @@ def AtomEditorComponents_Mesh_AddedToEntity():
         Report.result(Tests.creation_redo, mesh_entity.exists())
 
         # 5. Set Mesh component asset property
-        model_path = os.path.join('Objects', 'sphere_5lods_fbx_psphere_base_1.azmodel')
+        model_path = os.path.join('Objects', 'sphere_5lods_fbx_psphere_base.azmodel')
```

Comment:
```suggestion
        model_path = os.path.join('Objects', 'sphere_5lods.azmodel')
```

Reply:
according to the test
"# Update mesh asset to a model with 5 LOD materials"
does sphere_5lods.azmodel have multiple LOD materials?

Reply:
there is a `sphere.azmodel` not a `sphere_5lods.azmodel`

Reply:
https://github.com/o3de/o3de/blob/development/AutomatedTesting/Objects/sphere_5lods.fbx

Reply:
for this mesh test you could probably use the sphere.fbx, but sphere_5lods.fbx should work without issue, it was working without issue. the material test is utilizing LOD materials though so it would have to be redesigned to use a different asset.

Reply:
This test validates the Lod Type settings, including setting the lod override to a value that is > 0, and the sphere.fbx only has 1 lod, so we should also use https://github.com/o3de/o3de/blob/development/Gems/Atom/TestData/TestData/Objects/ModelHotReload/sphere_5lods.fbx for this test.

Reply:
As discussed offline, the testdata asset is failing locally for some reason. We can follow up and make that change, and in the meantime this test should not be blocking dev from being green, so submitting the change that renames it from ..._base_1 to ..._base is a good interim solution.